### PR TITLE
12191

### PR DESCRIPTION
**Pull request by vipullakhani**

### CODE REVIEW

File: mi/dataset/driver/ctdmo_ghqr/sio/ctdmo_ghqr_sio_ct_recovered_driver.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""
+@package mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered
+@file mi-dataset/mi/dataset/driver/ctdmo_ghqr/sio/ctdmo_ghqr_sio_ct_recovered_driver.py
```
Comment (renegelinas) [inline mi/dataset/driver/ctdmo_ghqr/sio/ctdmo_ghqr_sio_ct_recovered_driver.py]: package should be mi.dataset.driver.ctdmo_ghqr.sio

File: mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02001_CT.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 header:
   particle_object: CtdmoGhqrRecoveredHostInstrumentDataParticle
-  particle_type: ctdmo_ghqr_instrument_host_recovered
+  particle_type: ctdmo_ghqr_instrument_recovered
```
Comment (renegelinas) [inline mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02001_CT.yml]: I thought when we researched the preloaded database we found the ctdmo_ghqr_sio_mule_instrument telemetered data stream to be what you needed.  Unless you investigated the preloaded database further and found ctdmo_ghqr_instrument_recovered to be closer.

File: mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02004_CT.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 header:
   particle_object: CtdmoGhqrRecoveredHostInstrumentDataParticle
-  particle_type: ctdmo_ghqr_instrument_host_recovered
+  particle_type: ctdmo_ghqr_instrument_recovered
```
Comment (renegelinas) [inline mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02004_CT.yml]: dito

File: mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02100_CT.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 header:
   particle_object: CtdmoGhqrRecoveredHostInstrumentDataParticle
-  particle_type: ctdmo_ghqr_instrument_host_recovered
+  particle_type: ctdmo_ghqr_instrument_recovered
```
Comment (renegelinas) [inline mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02100_CT.yml]: dito

File: mi/dataset/parser/ctdmo_ghqr_sio.py [hunk 5]
```diff
@@ -313,7 +312,7 @@ class CtdmoGhqrRecoveredHostInstrumentDataParticle(DataParticle):
     """
     Class for generating Instrument Data Particles from Recovered data.
     """
-    _data_particle_type = DataParticleType.REC_CT_HOST_PARTICLE
+    _data_particle_type = DataParticleType.REC_CT_PARTICLE
```
Comment (renegelinas) [inline mi/dataset/parser/ctdmo_ghqr_sio.py]: dito

File: mi/dataset/parser/test/test_ctdmo_ghqr_sio.py [hunk 6]
```diff
@@ -191,7 +191,7 @@ def test_rec_co_and_ct_big_giant_input(self):
             for particle in result:
                 if particle._data_particle_type == DataParticleType.REC_CO_PARTICLE:
                     particle_co.append(particle)
-                if particle._data_particle_type == DataParticleType.REC_CT_HOST_PARTICLE:
+                if particle._data_particle_type == DataParticleType.REC_CT_PARTICLE:
```
Comment (renegelinas) [inline mi/dataset/parser/test/test_ctdmo_ghqr_sio.py]: dito for the similar changes in this file.

File: mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02100_CT.yml [hunk 7]
```diff
@@ -1,6 +1,10 @@
 header:
   particle_object: CtdmoGhqrRecoveredHostInstrumentDataParticle
-  particle_type: ctdmo_ghqr_instrument_host_recovered
+<<<<<<< HEAD
```
Comment (danmergens) [inline mi/dataset/driver/ctdmo_ghqr/sio/resource/CTD02100_CT.yml]: finish merge

**Pull request merged**